lwt/data_channel.ml

```diff
@@ -67,7 +67,7 @@ let of_fd fd ~seekable = (if seekable then of_seekable_fd else of_unseekable_fd)
 
 let sslctx =
   Ssl.init ();
-  Ssl.create_context Ssl.SSLv23 Ssl.Client_context
+  Ssl.create_context Ssl.TLSv1_2 Ssl.Client_context
```

Comment:
Is there a mode that can use TLS 1.3 as well as 1.2? `SSLv23` does that but also fall back even further if OpenSSL is compiled with support for legacy protocols

Reply:
nevermind, openssl 1.0.2 doesn't support tls 1.3 :(